### ISSUE TITLE
Removed git diff from .travis.yml

### ISSUE THREAD

**Issue by dougbtv** [dougbtv/whereabouts#24]
I removed this line in #23 -- it was failing on me, and I figured it "should have" worked with the changes that were made in that PR, so I removed it for the time being.

But, I wanted to track it. I'm guessing it's intended to fail if code generation wasn't performed? It should probably get put back together.

```
  - ./hack/generate-code.sh
  # - bash -c 'git diff pkg/ doc/ && git diff-index --quiet HEAD -- pkg/ doc/'
```

**Comment by crandles** [comment]
Correct, that was the intention with this. There are some deepcopy `*.go` files and a CRD yaml generated that update depending on the `IPPool` struct field / comments -- this validates the code that is checked in matches the latest regenerated artifacts.
